app/src/components/pages/collectionsPage/collectionsPage.tsx

```diff
@@ -159,11 +160,12 @@ export function CollectionsPage({ data, params, renderCollections }) {
             onChange: (e) => handleSearchChange(e),
           }}
           onSubmit={handleSearchSubmit}
-          labelText={""}
+          labelText="Search Collections by title"
+          aria-label="Search Collections by title"
```

Comment:
major nit but should collections be capitalized

Reply:
Hmm good point. I don't think you need the aria-label, the `labelText` should cover that.